utility/src/test/java/org/oppia/android/util/locale/DisplayLocaleImplTest.kt

```diff
@@ -159,6 +160,15 @@ class DisplayLocaleImplTest {
     assertThat(formattedString).containsMatch("[,.]")
   }
 
+  @Test
+  fun testToHumanReadableString_forInt_returnsStringWithExactNumber() {
```

Comment:
What about values larger than 10,000? It would be nice to verify if it automatically adds commas/periods (or not), based on the locale.

Reply:
A test has been added for locales that format numbers differently